standard/sections/clause_7_normative_text.adoc

```diff
@@ -8,7 +8,7 @@ A WNM provides notification metadata about data or metadata published to WIS2, i
 
 WIS2 Notification Message is an extension of the OGC API - Features standard [3] and shall be encoded in GeoJSON (RFC7946 [4]).
 
-=== Conformance Class Core
+=== Core Conformance Class
```

Comment:
Should there be consistency with the first sentence?
"This Core Conformance Class..."

Reply:
The heading is usually `Conformance Class "X"` or `Requirements Class "X"`.  I've changed to quoting the conformance class per OGC ModSpec.